src/azure-cli/azure/cli/command_modules/acr/task.py

```diff
@@ -982,3 +990,34 @@ def _get_trigger_event_list_patch(cmd,
             if SourceTriggerEvent.pullrequest.value in source_trigger_events:
                 source_trigger_events.remove(SourceTriggerEvent.pullrequest.value)
     return source_trigger_events
+
+def prepare_context_path(source_location, client_registries, registry_name, resource_group_name):
+    if source_location.lower() == NULL_SOURCE_LOCATION:
```

Comment:
Add null check. Otherwise, it will fail when there's no context. This function in run doesn't need the check because the context is mandatory.